packages/cli-platform-apple/src/commands/runCommand/installApp.ts

```diff
@@ -0,0 +1,111 @@
+import child_process from 'child_process';
+import {CLIError, logger} from '@react-native-community/cli-tools';
+import {IOSProjectInfo} from '@react-native-community/cli-types';
+import chalk from 'chalk';
+import {getBuildPath} from './getBuildPath';
+import {getBuildSettings} from './getBuildSettings';
+import path from 'path';
+import {ApplePlatform} from '../../types';
+
+function handleLaunchResult(
+  success: boolean,
+  errorMessage: string,
+  errorDetails = '',
+) {
+  if (success) {
+    logger.success('Successfully launched the app');
+  } else {
+    logger.error(errorMessage, errorDetails);
+  }
+}
+
+type Options = {
+  buildOutput: any;
+  xcodeProject: IOSProjectInfo;
+  mode: string;
+  scheme: string;
+  target?: string;
+  udid?: string;
+  binaryPath?: string;
+  platform?: ApplePlatform;
+};
+
+export default async function installApp({
```

Comment:
Should this be split into separate `installApp` and `openApp`? It's a bit misleading that `installApp` does both.

In particular, "installing" the app isn't necessary for macOS apps since we can just run `open MyApp.app` directly.